quiche/src/lib.rs

```diff
@@ -4221,6 +4364,17 @@ impl<F: BufFactory> Connection<F> {
                 // be bundled considering the buffer capacity only, and not the
                 // available cwnd.
                 if push_frame_to_pkt!(b, frames, frame, left) {
+                    self.largest_pkt_acked =
+                        pkt_space.recv_pkt_need_ack.last().unwrap();
```

Comment:
adding a comment that this never panics because .len() > 0 is a condition above may help future reviewer.